glfw/src/main.zig

```diff
@@ -89,6 +92,7 @@ pub inline fn init(hints: InitHints) Error!void {
 ///
 /// Possible errors include glfw.Error.PlatformError.
 ///
+// TODO: Should this remark be removed? Or should we allow this function to be called before init?
 /// remark: This function may be called before glfw.init.
```

Comment:
Good question.. I would say that in general someone calling terminate without a prior init is likely to be a bug in their code, so I'd err on the side of removing this remark instead.